.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
```

Comment:
By the way, why is this false?

Reply:
This was auto-generated by EditorConfig extension. We should update this.